src/device/pf_cmrdr.c

```diff
@@ -51,7 +51,7 @@ int pf_cmrdr_rm_read_ind(
    uint16_t                start_pos = 0;
    uint8_t                 iocs[255];              /* Max possible array size */
    uint8_t                 iops[255];              /* Max possible array size */
-   uint8_t                 subslot_data[1500];     /* Max possible array size */
+   uint8_t                 subslot_data[PF_FRAME_BUFFER_SIZE];     /* Max possible array size */
```

Comment:
For embedded systems, it is generally not a great idea to have huge arrays on the stack. However that can be addressed in another commit.